### ISSUE TITLE
Allow moving Groups

### ISSUE THREAD

**Issue by zurqlisp** [timer-machine/timer-machine-android#46]
I need it to replace many "Study" timers.with pomodoro-like groups

**Comment by DeweyReed** [comment]
Good idea! Because it involves a lot of work, I'll implement it when I renovate the whole editing screen.
